### DISCUSSION TITLE
Why "Multiple handoffs detected only the first is executed"? (dev10)

### DISCUSSION THREAD

**Discussion by imhuay** [microsoft/autogen#4638] [Q&A] [answered]
I am using the Swarm mode of the newest api dev10.
My planner returen multi FunctionExecutionResult, but only the first one execute. 
If I want to execute all of them and aggregate the results, which mode should I use?

**Comment by ekzhu** [comment, marked as answer]
Swarm is sequential only, we are planning to disable parallel tool call for handoffs to avoid this warning. 

For parallel processing, for now, you can take a look at the Core API examples: https://microsoft.github.io/autogen/dev/user-guide/core-user-guide/design-patterns/index.html
